docs/consume-packages/consuming-packages-authenticated-feeds.md

```diff
@@ -5,6 +5,7 @@ author: nkolev92
 ms.author: nikolev
 ms.date: 12/22/2023
 ms.topic: conceptual
+ms.custom: sfi-ropc-nochange

```

Comment:
May I know what `sfi-ropc-nochange` stands for? How is it connected to `GA and `Images`?